### PR TITLE
FIP-32438: fix deprecation and bump a gem version

### PR DESCRIPTION
**Pull request by pfeffer**

### CODE REVIEW

File: lib/marqeta/api_object.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def created_time
     attr_accessor :attributes_hash
 
     def symbolized_attributes_hash
-      Hash[attributes_hash.map { |k, v| [k.to_sym, v] }]
+      Hash[attributes_hash.to_h.map { |k, v| [k.to_sym, v] }]
```
Comment (dubroe) [inline lib/marqeta/api_object.rb:56]: What's the reason for this change?

Comment (dubroe) [reply lib/marqeta/api_object.rb:56]: Oh duh. I see why.

Comment (pfeffer) [reply lib/marqeta/api_object.rb:56]: the deprecation is in the ticket description 😬
